src/Tizen.NUI/src/public/Window.cs

```diff
@@ -1117,6 +1117,18 @@ public void SetAvailableOrientations( List<Window.WindowOrientation> orientation
             if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
         }
 
+        /// <summary>
+        /// Get native window ID
+        /// </summary>
+        /// <returns>native window ID</returns>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public int GetWindowId()
```

Comment:
We already provides GetNativeHandle. I think GetNativeId is better.

Reply:
Hello, I fixed it as following comments. please review. thank you.